.github/workflows/cmake.yml

```diff
@@ -0,0 +1,15 @@
+name: NetLRTS Linux CMake
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: build
+      run: ./buildcmake LIBS netlrts-linux-x86_64 -g -j4 --with-production
+    - name: test
+      run: make -C netlrts-linux-x86_64/tests test OPTS="-g" TESTOPTS="++local +isomalloc_sync"
```

Comment:
Should this be `make -C netlrts-linux-x86_64/tests all OPTS="-g" -j4 && make -C netlrts-linux-x86_64/tests test OPTS="-g" TESTOPTS="++local +isomalloc_sync"`?

Reply:
Fixed. I removed the second 'OPTS'.